lucid/misc/io/scoping.py

```diff
@@ -20,10 +20,13 @@ def set_io_scopes(scopes):
 
 
 @contextmanager
-def io_scope(path):
+def io_scope(path, absolute=False):
```

Comment:
as discussed oob: let's use `replace_current_scopes` or similar as the kwarg. :-)